python/networkx.cpp

```diff
@@ -561,7 +561,29 @@ void ukv::wrap_networkx(py::module& m) {
         "A DegreeView with the number outgoing edges for each Vertex.");
     g.def(
         "size",
-        [](py_graph_t& g) { return g.relations_attrs.size(); },
+        [](py_graph_t& g, std::string weight) {
+            if (!weight.size())
+                return g.ref().number_of_edges();
+
+            std::size_t size = 0;
+            auto stream = g.ref().edges(ukv_vertex_source_k).throw_or_release().begin();
```

Comment:
Do we have to iterate through edges like this, or can we simply iterate through just the document collection of edge attributes, extracting only one field from many documents?

Reply:
When the edge does not have any attributes document collection does not contains that edge id, but we must calculate those edges with weight=1

Reply:
Are those attributes supposed to be just strings? According to [your last commit](https://github.com/unum-cloud/ukv/pull/323/commits/1648cb3ef09faeea4bc9707d90169ba665192a61) if we have failed to parse the string we increment by zero - negating your previous statement.

Reply:
> This

In Networkx I get "TypeError: unsupported operand type(s) for +: 'int' and 'str' "  when I pass string as  weight attribute...I will fix it in our implementation